src/components/login/LoginPage.tsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import {StyleSheet, View, TextInput, Text, TouchableOpacity, Alert, Image, Pressable} from 'react-native';
+
+const LoginPage = () => {
+  const [text, onChangeText] = React.useState(null);
+  const [passwordSecured, setPasswordSecured] = React.useState(true);
+
+  return (
+    <View style={styles.containerBox}>
+        <View style={styles.containerImage}>
+          <Image resizeMode='contain' style={styles.imageLogin} source={require('../../../assets/images/login.png')} />
+        </View>
+        <View style={styles.inputView}>
+          <Image resizeMode='contain' style={styles.imageInput} source={require('../../../assets/images/email.png')} />
+          <TextInput
+            style={styles.input}
+            onChangeText={onChangeText}
+            value={text}
+            placeholder={"Enter your Email"}
+            placeholderTextColor={"#a8a8a8"}
+          />  
+        </View>
+        <View style={styles.inputView}>
+          <TextInput
+            style={styles.input}
+            secureTextEntry={passwordSecured}
+            textContentType='password'
+            placeholder={"Password"}
+            placeholderTextColor={"#a8a8a8"}
+          />        
+          <TouchableOpacity
+            style={styles.passwordIcon}
+            onPress={() => {
+              setPasswordSecured(!passwordSecured);
+            }}>
+          <Image resizeMode='contain' style={styles.imageInput} source={require('../../../assets/images/eye.png')} />
```

Comment:
Please avoid to use strict strings for path.
Create an index file  in assets/image folder like:

```
import email from './email.png'
import eye from './eye.png'
import login from './login.png'
// ....

export {
email,
eye,
login
}
```

and in the file you can use like:
```
import { email, eye, login } from '../../../assets/images'
// ...
<Image resizeMode='contain' style={styles.imageInput} source={eye} />
```